mobile/src/components/chat-conversation/components/messages-list/messages-list.tsx

```diff
@@ -51,6 +68,14 @@ const MessagesList: FC<Props> = ({ currentUserId, messages }) => {
         )}
         inverted
         contentContainerStyle={styles.messageList}
+        getItemLayout={(
+          _,
+          index,
+        ): { length: number; offset: number; index: number } => ({
+          length: renderMessages.length,
+          offset: renderMessages.length * index,
+          index,
+        })}
```

Comment:
Won't scroll work without it? if not, rename `_` variable to understand what do we have as first parameter in the future, like `_item`

Reply:
In the documentation for SectionList I saw the notice, and decide use getItemLayout ( without it didn't work )
![image](https://user-images.githubusercontent.com/82529236/190329836-930ff31b-9c64-4c42-9286-d347764f7465.png)